src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -45,7 +43,7 @@ internal static partial class RoslynExtensions {
 			return fullyQualifiedName;
 		}
 
-		public static bool IsNullOrErrorType( this ITypeSymbol symbol ) {
+		public static bool IsNullOrErrorType( [NotNullWhen( false )] this ITypeSymbol? symbol ) {

```

Comment:
Own PR, if you could

Reply:
> Own PR, if you could

https://github.com/Brightspace/D2L.CodeStyle/pull/799/files